src/drivers/frsky_telemetry/sPort_data.c

```diff
@@ -302,13 +317,31 @@ void sPort_send_GPS_CRS(int uart)
 	/* send course */
 
 	/* convert to 30 bit signed magnitude degrees*6E5 with MSb = 1 and bit 30=sign */
-	uint32_t iYaw = 100 * global_pos->yaw;
+	int32_t iYaw = local_pos->yaw * 18000.0f / M_PI_F;
```

Comment:
@LorenzMeier Should I return this back to using global position so we can merge the rest? I definately think we should fix the GPS and Altitude errors